firefox_media_tests/playback/netflix/test_basic_playback.py

```diff
@@ -0,0 +1,66 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from media_test_harness.testcase import MediaTestCase
+from firefox_media_tests.playback.video_playback_tests_mixin import (
+    VideoPlaybackTestsMixin
+)
+
+
+class TestBasicNetflixPlayback(MediaTestCase,
+                               VideoPlaybackTestsMixin):
+
+    def setUp(self):
+        super(TestBasicNetflixPlayback, self).setUp()
+        self.set_eme_prefs()
+        assert(self.check_eme_prefs())
+
+    def set_eme_prefs(self):
+        with self.marionette.using_context('chrome'):
+            self.prefs.set_pref('media.gmp.trial-create.enabled', False)
+
+    def check_eme_prefs(self):
+        prefs_ok = True
+        with self.marionette.using_context('chrome'):
+            mediasource_enabled = self.prefs.get_pref(
+                'media.mediasource.enabled'
+            )
+            if mediasource_enabled is None or not mediasource_enabled:
+                self.logger.info('media.mediasource.enabled not True.')
+                prefs_ok = False
+
+            mediasource_mp4_enabled = self.prefs.get_pref(
+                'media.mediasource.mp4.enabled'
+            )
+            if mediasource_mp4_enabled is None or not mediasource_mp4_enabled:
+                self.logger.info('media.mediasource.mp4.enabled not True.')
+                prefs_ok = False
+
+            gmp_eme_adobe_enabled = self.prefs.get_pref(
+                'media.gmp-eme-adobe.enabled'
+            )
+            if gmp_eme_adobe_enabled is None or not gmp_eme_adobe_enabled:
+                self.logger.info('media.gmp-eme-adobe.enabled not True.')
+                prefs_ok = False
+
+            gmp_eme_adobe_version = self.prefs.get_pref(
+                'media.gmp-eme-adobe.version'
+            )
+            if gmp_eme_adobe_version is None:
+                self.logger.info('media.gmp-eme-adobe.version does not exist.')
+                prefs_ok = False
+            else:
+                try:
+                    if gmp_eme_adobe_version < 1:
+                        self.logger.info(
+                            'media.gmp-eme-adobe.version has a non-positive value.'
+                        )
+                        prefs_ok = False
+                except TypeError:
+                    self.logger.info(
+                        'media.gmp-eme-adobe.version is not an integer value.'

```

Comment:
Ok, but nothing in the try block will raise if it is a float. It won't even raise if `gmp_eme_adobe_version' is something like True because `True < 1` works fine. At least include the value of `gmp_eme_abone_version` in the log message to help with future debugging.